xds/src/main/java/io/grpc/xds/EnvoyProtoData.java

```diff
@@ -1806,4 +1806,6 @@ EndpointLoadMetricStats build() {
       }
     }
   }
+
+  static final String TRANSPORT_SOCKET_NAME_TLS = "envoy.transport_sockets.tls";
```

Comment:
nit: could you put this to the top of this class (like line 1716).

Reply:
Done